src/packages/@ncigdc/uikit/Tooltip/GlobalTooltip.js

```diff
@@ -11,12 +12,16 @@ class GlobalTooltip extends Component {
     window.removeEventListener('mousemove', this.moveTooltip);
   }
 
-  moveTooltip = event => {
-    this.globalTooltip.style.left = `${event.pageX}px`;
-    this.globalTooltip.style.top = `${event.pageY -
-      this.globalTooltip.offsetHeight -
-      15}px`;
-  };
+  moveTooltip = _.throttle(event => {
+    const left = event.pageX;
+    const top = event.pageY;
+    setTimeout(() => {
```

Comment:
This either fixes it or makes it edgier to the point of being undifferentiable from being fixed :p

I'm not sure if the position would be correct with the current global tooltip solution if the listener is added after componentDidUpdate. The mouse move event may happen before that lifecycle hook is called